snuba/settings/__init__.py

```diff
@@ -241,13 +241,7 @@ class RedisClusters(TypedDict):
 
 # The migration groups that can be skipped are listed in OPTIONAL_GROUPS.
 # Migrations for skipped groups will not be run.
-SKIPPED_MIGRATION_GROUPS: Set[str] = {
-    "querylog",
-    "profiles",
-    "functions",
-    "test_migration",
-    "search_issues",
-}
+SKIPPED_MIGRATION_GROUPS: Set[str] = set()
```

Comment:
Should we just remove this?

Reply:
Updated!

Reply:
Rebased master, we just added `spans` migration group which will still use SKIPPED_MIGRATIONS_GROUPS for now.

Reply:
why not just set its readiness state since it's new?

Reply:
Unfortunately, readiness states do not capture the current roll out process for `spans` which is a unique case.